CodeEditor/Sources/CodeEditor/UXCodeTextView.swift

```diff
@@ -290,12 +291,24 @@ final class UXCodeTextView: UXTextView, HighlightDelegate, UIScrollViewDelegate
         if let font = theme.codeFont, font !== self.font { self.font = font }
         guard theme.codeFont?.pointSize != newSize else { return true }
 
-        theme.codeFont       = theme.codeFont?      .withSize(newSize)
+        theme.codeFont = theme.codeFont?      .withSize(newSize)
         theme.boldCodeFont   = theme.boldCodeFont?  .withSize(newSize)
         theme.italicCodeFont = theme.italicCodeFont?.withSize(newSize)
+        if let newTheme {
+            themeName = newTheme
+        }
         return true
     }
     
+    func changeFontSize(size: CGFloat) {
+        if let theme = highlightr?.theme {
+            theme.codeFont = theme.codeFont?      .withSize(size)
+            theme.boldCodeFont   = theme.boldCodeFont?  .withSize(size)
+            theme.italicCodeFont = theme.italicCodeFont?.withSize(size)
```

Comment:
I would suggest either removing the space padding or making it so that all of the dots align

```suggestion
            theme.codeFont = theme.codeFont?            .withSize(size)
            theme.boldCodeFont   = theme.boldCodeFont?  .withSize(size)
            theme.italicCodeFont = theme.italicCodeFont?.withSize(size)
```